pages/desktop/user.py

```diff
@@ -165,8 +166,78 @@ def clear_field(self):
 
 class MyCollections(Base):
 
-    _header_locator = (By.CSS_SELECTOR, ".primary > header > h2")
+    _header_locator = (By.CSS_SELECTOR, '.primary > header > h2')
+    _my_favorites_locator = (By.CSS_SELECTOR, '.other-categories ul:nth-child(3) li:nth-child(3)')
 
     @property
     def my_collections_header_text(self):
         return self.selenium.find_element(*self._header_locator).text
+
+    def click_my_favorites(self):
+        self.selenium.find_element(*self._my_favorites_locator).click()
+        return MyFavorites(self.base_url, self.selenium)
+
+
+class MyFavorites(Base):
+
+    _page_title = 'My Favorite Add-ons :: Collections :: Add-ons for Firefox'
+    _header_locator = (By.CSS_SELECTOR, "h2.collection > span")
+
+    _edit_collection_locator = (By.CLASS_NAME, 'edit')
+    _addon_locator = (By.CSS_SELECTOR, '.separated-listing .item')
+    _add_on_name_locator = (By.CSS_SELECTOR, '.item h3 a')
+
+    def edit_collection(self):
+        self.selenium.find_element(*self._edit_collection_locator).click()
+        return EditCollection(self.base_url, self.selenium)
+
+    @property
+    def my_favorites_header_text(self):
+        return self.selenium.find_element(*self._header_locator).text
+
+    @property
+    def add_ons(self):
+        return [AddOn(self.base_url, self.selenium, el) for el in
+                self.selenium.find_elements(*self._addon_locator)]
```

Comment:
I wasn't expecting you to implement this, but thanks for giving it a go! We can't actually reuse `AddOn` from `pages.desktop.regions.addon` because that's a different region. Here I would suggest a nested class named `AddOn` as the region is specific to the collection page.